src/screens/joiningscreen.cpp

```diff
@@ -1096,8 +1099,8 @@ void WzJoiningGameScreen_HandlerRoot::processOpenConnectionResult(size_t connect
 
 	if (NETgetEnableTCPNoDelay())
 	{
-		// Enable TCP_NODELAY
-		socketSetTCPNoDelay(*client_transient_socket, true);
+		// Disable use of Nagle Algorithm for the TCP socket (i.e. enable TCP_NODELAY option in case of TCP transport)
+		client_transient_socket->useNagleAlgorithm(true);
```

Comment:
Shouldn’t this be `useNagleAlgorithm(false)`?

Reply:
Fixed.